mozci/console/commands/push.py

```diff
@@ -28,6 +29,37 @@ def handle(self):
         self.line(tabulate(table, headers=["Label", "Result"]))
 
 
+def classify_commands_pushes(branch, from_date, to_date, rev):
+    if not (bool(rev) ^ bool(from_date or to_date)):
+        return (
```

Comment:
Instead of returning the error as a second return value, let's raise an exception and catch it in the caller